xtas/tests/test_corenlp.py

```diff
@@ -61,7 +61,7 @@ def test_lemmatize_unicode():
                   annotators=['tokenize', 'ssplit', 'pos', 'lemma'])
     saf = stanford_to_saf(lines)
     assert_equal({t['lemma'] for t in saf['tokens']},
-                 {'Cesar', 'hit', 'Hovik'})
```

Comment:
`{}` is set builder syntax, so the order is considered irrelevant. From the looks of stanford_to_saf output, the order shouldn't matter because each token carries an offset.